build.gradle

```diff
@@ -118,6 +185,12 @@ dependencies {
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${junit5Version}")
     testImplementation("org.opensearch.test:framework:${opensearchVersion}")
     testRuntimeOnly("org.junit.platform:junit-platform-launcher:${junitPlatform}")
+
+    configurations.all {
+        resolutionStrategy.force("jakarta.json:jakarta.json-api:2.1.1")
```

Comment:
just a nit, lets keep the version numbers defined at the start of the dependencies section. Makes maintaining/upgrading dependencies easier in the future.